ww

```diff
@@ -16,7 +16,14 @@ activate() {
     errorOut "use 'source ww $1'"
   fi
 
-  workspace="${WW_DEFAULT_PATH}/${1}"
+  local workspace="${1}"
+
+  # Checks it provided workspace fits as a relative path.
+  # If not, then consider it an absolute path.
+  if [ -d "${WW_DEFAULT_PATH}/${workspace}" ]; then
```

Comment:
I get that on linux there is not multiple workspace roots (like windows).

Not a problem.
I think having multiples roots not really nice =)